Userland/Libraries/LibC/unistd.cpp

```diff
@@ -975,8 +975,41 @@ int unveil(char const* path, char const* permissions)
 // https://pubs.opengroup.org/onlinepubs/9699919799/functions/getpass.html
 char* getpass(char const* prompt)
 {
-    dbgln("FIXME: getpass('{}')", prompt);
-    TODO();
+    int tty = open("/dev/tty", O_RDWR | O_NOCTTY | O_CLOEXEC);
+
+    if (tty < 0)
+        return nullptr;
+
+    struct termios backup, noecho;
```

Comment:
Please declare only one variable per line, and zero init these structures with `{}`.